src/replication/replicator.rst

```diff
@@ -127,8 +127,11 @@ The replication job will also appear in
       }
 
 ``_scheduler/jobs`` shows more information such as a detailed history of
```

Comment:
grammatical improvement (I know this one existed before your edit, but we may as well correct it at the same time):

```suggestion
``_scheduler/jobs`` shows more information, such as a detailed history of
```